lib/sub_manager.js

```diff
@@ -39,8 +39,9 @@ SubsManager.prototype.subscribe = function() {
   }
 };
 
-SubsManager.prototype._addSub = function(args) {
+SubsManager.prototype._addSub = function() {
   var self = this;
+  var args = _.toArray(arguments);
   var hash = EJSON.stringify(args);
   var subName = args[0];
```

Comment:
This returns undefined, args is the first arguments. To be able to index the arguments. We need args = _.toArray(arguments)